From 65cdb80525b12410010e1d4f02bf07b3fb1bac63 Mon Sep 17 00:00:00 2001
From: Jingoo Han <jg1.han@samsung.com>
Date: Fri, 6 Jun 2014 14:35:58 -0700
Subject: [PATCH 653/687] UPSTREAM: rtc: rtc-palmas: make of_device_id array
 const

(cherry pick from commit 7abe3f5404fae84cecd851e140a2218ba9d24e4f)

Make of_device_id array const, because all OF functions handle it as
const.

Signed-off-by: Jingoo Han <jg1.han@samsung.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Linus Torvalds <torvalds@linux-foundation.org>
Signed-off-by: Mark Salyzyn <salyzyn@google.com>
Bug: 26346842
Change-Id: Idb8b070b72a77c80df6c7f1aae92f803243295c3
---
 drivers/rtc/rtc-palmas.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/rtc/rtc-palmas.c b/drivers/rtc/rtc-palmas.c
index fffb7d3..ccc8fc6 100644
--- a/drivers/rtc/rtc-palmas.c
+++ b/drivers/rtc/rtc-palmas.c
@@ -353,7 +353,7 @@ static const struct dev_pm_ops palmas_rtc_pm_ops = {
 };
 
 #ifdef CONFIG_OF
-static struct of_device_id of_palmas_rtc_match[] = {
+static const struct of_device_id of_palmas_rtc_match[] = {
 	{ .compatible = "ti,palmas-rtc"},
 	{ },
 };
-- 
1.9.1

